src/gnutls.mk

```diff
@@ -28,8 +28,8 @@ define $(PKG)_BUILD
         --enable-local-libopts \
         --with-included-libtasn1 \
         --with-libregex-libs="-lgnurx" \
-        --without-p11-kit \
         --disable-silent-rules \
+		--with-unbound-root-key-file="C:\\\\Program Files\\\\Unbound\\\\root.key"
```

Comment:
1. Please fix alignment.
2. About "C:\\\\Program Files\\\\Unbound\\\\root.key" path... Can you provide more information on this path? Can we avoid hardcoding it in the build rules?